tests/test_dagmc_stats.py

```diff
@@ -213,10 +213,53 @@ def test_get_roughness(self):
         my_core = test_env[1]['core']
         native_ranges = test_env[1]['native_ranges']
         
-        roughness = ds.get_roughness(my_core, native_ranges)
+        roughness = ds.get_roughness(my_core, native_ranges).values()
         exp = 8
         obs = len(roughness)
         self.assertEqual(exp, obs)
         
         exp = [0,0,0,0,0,0,0,0]
         np.testing.assert_allclose(roughness, exp, atol = 1e-04)
+        
+        
+    def test_get_avg_roughness(self):
+        """Tests part of the avg_roughness function
+        """
+        # single_cube
+        my_core = test_env[1]['core']
+        native_ranges = test_env[1]['native_ranges']
+        roughness = ds.get_roughness(my_core, native_ranges)
+        geom_dim = test_env[1]['dagmc_tags']['geom_dim']
+        
+        exp = 0
+        obs = ds.avg_roughness(my_core, roughness, geom_dim)
+        self.assertAlmostEqual(exp, obs)
+        
+        # pyramid_2
+        my_core = test_env[2]['core']
+        native_ranges = test_env[2]['native_ranges']
+        roughness = ds.get_roughness(my_core, native_ranges)
+        geom_dim = test_env[2]['dagmc_tags']['geom_dim']
+        
+        gc_top = 2.0/3*np.pi
+        gc_bottom = 5.0/6*np.pi
+        d_bottom = [1/np.tan(np.pi/3),
+                    0.5*(1/np.tan(np.pi/3)+1/np.tan(np.pi/4)), 0]
+        lr_bottom=[]
+        lr_top = np.abs(gc_top - gc_bottom)
+        lr_bottom.append(np.abs(gc_bottom - \
+                    (d_bottom[0]*gc_top+2*d_bottom[1]*gc_bottom) \
+                    /(d_bottom[0]+2*d_bottom[1])))
+        lr_bottom.append(np.abs(gc_bottom - \
+                    (d_bottom[0]*gc_top+2*d_bottom[1]*gc_bottom+d_bottom[2]*gc_bottom) \
+                    /(d_bottom[0]+2*d_bottom[1]+d_bottom[2])))
+        s_top = 4*25/4*np.sqrt(3)
+        s_bottom = [12.5+2*25/4*np.sqrt(3), 25+2*25/4*np.sqrt(3)]
+        num = lr_top*s_top/3 + \
+                    2*lr_bottom[0]*s_bottom[0]/3 + \
+                    2*lr_bottom[1]*s_bottom[1]/3
+        denom = 4*25/4*np.sqrt(3)+25
```

Comment:
there are a lot of magic numbers in this section. I am mostly wondering what the 25 number is here?

Reply:
some comments for what each of these calculations is doing will be helpful to future developers in the event that this test fails.

Reply:
25 is the area of the bottom surface which is a 5*5 square